### DISCUSSION TITLE
.ts files in config directory

### DISCUSSION THREAD

**Discussion by mhendu** [fuatakgun/eufy_security#266] [Q&A]
I saw someone else asked about this and the recommendation was to set up RTSP, but I can't find anywhere in the app to enable RTSP (I have installed the RTSP add-on but not apparent to me how to get the devices to stream via RTSP). I have the doorbell cam and the floodlight cam. I'm getting dozens of these .ts files dropped into my config directory and it's slowing down file editor. How can I prevent these files from being saved in this directory?

**Comment by fuatakgun** [comment]
Check step 10 here: https://github.com/fuatakgun/eufy_security#63-installing-integration

Screenshot is not up-to-date but you will get it

**Comment by mhendu** [comment]
Perfect, thank you!
